sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md

```diff
@@ -2,8 +2,16 @@
 
 ## 1.0.0-preview.3 (Unreleased)
 
-### Breaking changes
+### Renames
 
+- `FormRecognizerError.Code` renamed to `FormRecognizerError.ErrorCode`.
+- `FormTrainingClient.GetModelInfos` renamed to `FormTrainingClient.GetCustomModels`.
+- Parameter `useLabels` in `FormTrainingClient.StartTraining` renamed to `useTrainingLabels`.
+- Parameter `trainingFiles` in `FormTrainingClient.StartTraining` renamed to `trainingFilesUri`.
+
+### Other breaking changes
```

Comment:
Let me know if this sounds good or not. Renamed it to `Other breaking changes` because `Renames` are breaking changes, after all.